src/com/zone5cloud/retrofit/core/apis/UserAPI.java

```diff
@@ -72,4 +72,12 @@
     /** Refresh a bearer token - get a new token if the current one is nearing expiry */
     @GET(Users.REFRESH_TOKEN)
     Observable<Response<OAuthToken>> refreshToken();
+
+    /** Returns password complexity */
+    @GET(Users.PASSWORD_COMPLEXITY)
+    Observable<Response<String>> passwordComplexity();
+
+    /** Reconfirm email */
+    @GET(Users.RECONFIRM)
+    Observable<Response<Void>> reconfirm(@Query("email") String email);
 }
```

Comment:
Make sure you have run tests for both of these new apis that hits the specialized staging server ("https://api-sp-staging.todaysplan.com.au") and verifies their responses.